yt/yt/library/auth_server/cypress_cookie_login.cpp

```diff
@@ -123,7 +135,7 @@ class TCypressCookieLoginHandler
                 HandleRegularRequest(rsp);
 
                 error = TError("No such user %Qlv or user has no password set", user) << error;
-                replyAndLogError(error, TString{user});
+                replyAndLogError(error, TString{user}, true);
```

Comment:
In C++ code we usually write `/*replyWithGenericError*/ true` in order to highlight what does this `true` mean.